Replicator/Worker.cc

```diff
@@ -336,4 +335,58 @@ namespace litecore::repl {
             output << "Coll=" << collIdx;
         }
     }
+
+    const std::unordered_set<slice> Options::kWhiteListOfKeysToLog{
+            // That is, they are supposed to be assigned to c4ReplicatorParameters.collections[i].optionsDictFleece
+            kC4ReplicatorOptionDocIDs,
+            kC4ReplicatorOptionChannels,
+            kC4ReplicatorOptionFilter,
+            kC4ReplicatorOptionFilterParams,
+            kC4ReplicatorOptionSkipDeleted,
+            kC4ReplicatorOptionNoIncomingConflicts,
+            // end of collection specific properties.
+
+            kC4ReplicatorCheckpointInterval,
+            kC4ReplicatorOptionRemoteDBUniqueID,
+            kC4ReplicatorOptionDisableDeltas,
+            kC4ReplicatorOptionDisablePropertyDecryption,
+            kC4ReplicatorOptionMaxRetries,
+            kC4ReplicatorOptionMaxRetryInterval,
+            kC4ReplicatorOptionAutoPurge,
+            kC4ReplicatorOptionAcceptParentDomainCookies,
+
+            // TLS options:
+            kC4ReplicatorOptionRootCerts,
+            kC4ReplicatorOptionPinnedServerCert,
+            kC4ReplicatorOptionOnlySelfSignedServerCert,
+
+            // HTTP options:
+            kC4ReplicatorOptionExtraHeaders,
+            kC4ReplicatorOptionCookies,
+            kC4ReplicatorOptionAuthentication,
+            kC4ReplicatorOptionProxyServer,
+
+            // WebSocket options:
+            kC4ReplicatorHeartbeatInterval,
+            kC4SocketOptionWSProtocols,
+            kC4SocketOptionNetworkInterface,
+
+            // BLIP options:
+            kC4ReplicatorCompressionLevel,
+
+            // [1]: Auth dictionary keys:
+            kC4ReplicatorAuthType,
+            kC4ReplicatorAuthUserName,
+            // kC4ReplicatorAuthPassword,
+            kC4ReplicatorAuthEnableChallengeAuth,
+            kC4ReplicatorAuthClientCert,
+            kC4ReplicatorAuthClientCertKey,
+            kC4ReplicatorAuthToken,
```

Comment:
Shall we log this?

Reply:
When find it out it's not okay to log, we can take it out from the white list easily.

Reply:
This info shouldn't be logged.

Reply:
okay, will remove it.